docs/pages/Cardinal/ecs.mdx

```diff
@@ -20,4 +20,4 @@ Systems are the heartbeat of ECS, encompassing functions that manipulate entitie
 
 ## Why Do We Need ECS?
 
-When used correctly, ECS unlocks a realm of organized code and seamless extensibility. Injecting new features into your game can be as straightforward as introducing a novel system that interacts with existing components. It's important to note that while ECS is a formidable tool, it isn't a universal solution. Certain game scenarios might find better suited solutions outside the ECS paradigm.
\ No newline at end of file
+When used correctly, ECS unlocks a realm of organized code and seamless extensibility. Injecting new features into your game can be as straightforward as introducing a novel system that interacts with existing components. It's important to note that while ECS is a formidable tool, it isn't a universal solution. Certain game scenarios might find better suited solutions outside the ECS paradigm.
```

Comment:
Maybe add links where the user can go to explore more about the ECS pattern.

Reply:
This is out of scope for this PR (focusing on Nakama docs).